packages/theming/android-theme/src/createAndroidTheme.ts

```diff
@@ -5,9 +5,9 @@ import { Theme, ThemeOptions } from '@fluentui-react-native/theme-types';
 
 export function createAndroidTheme(options: ThemeOptions = {}): ThemeReference {
   const themeRef = new ThemeReference({} as Theme, () => {
-    // Stub out HC on Android
+    // Stub out HC and darkElevated on Android
     const current =
-      options.appearance == 'dynamic' || options.appearance == 'highContrast'
+      options.appearance == 'dynamic' || options.appearance == 'highContrast' || options.appearance == 'darkElevated'
```

Comment:
Should these all be strict equality

Reply:
Yes, almost certainly. Probably no real difference in this case, but any day we can remove a `==` is a good day. I might even consider adding it as a linting rule.